modules/kafka-cluster/main.tf

```diff
@@ -1,6 +1,7 @@
 locals {
-  build_cluster_type_basic    = var.cluster_for_production ? [] : ["yes"]
-  build_cluster_type_standard = var.cluster_for_production ? ["yes"] : []
+  basic_cluster_type     = "basic"
```

Comment:
consider renaming to `cluster_type...` or `confluent_kafka_cluster_type...`

```
cluster_type_basic = "basic"
cluster_type_standard = "standard"
cluster_type_dedicated = "dedicated"
```

or ```
confluent_kafka_cluster_type_basic = "basic"
confluent_kafka_cluster_type_standard = "standard"
confluent_kafka_cluster_type_dedicated = "dedicated"
```

Reply:
incorporated this suggestion